posthog/management/commands/create_batch_export_from_app.py

```diff
@@ -0,0 +1,161 @@
+import datetime as dt
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+
+from posthog.batch_exports.service import backfill_export, create_batch_export
+from posthog.models.plugin import PluginConfig
+from posthog.temporal.client import sync_connect
+
+
+class Command(BaseCommand):
+    help = "Create a BatchExport from an existing Export app"
+
+    def add_arguments(self, parser):
+        """Add arguments to the parser."""
+        parser.add_argument(
+            "--plugin-config-id", type=int, help="The ID of the PluginConfig to use as a base for the new BatchExport"
+        )
+        parser.add_argument(
+            "--team-id",
+            type=int,
+            help="The ID of the team that owns the PluginConfig and where to create the BatchExport.",
+        )
+        parser.add_argument("--name", default=None, help="The name for the new BatchExport.")
```

Comment:
I think we should hardcode this part via map of `plugin_id -> batch export name` to avoid user-errors using this